tests81/Unit/Models/Enums/StatusEnum.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Backpack\CRUD\Tests81\Unit\Models\Enums;
+
+    enum StatusEnum: string
+    {
+        case DRAFT = 'DRAFT';
+        case PUBLISHED = 'PUBLISHED';
+
+        public static function getOptions()
+        {
+            return array_combine(array_column(self::cases(), 'name'), array_column(self::cases(), 'value'));
```

Comment:
```suggestion
            return array_column(self::cases(), 'value', 'name');
```

The resulting array will be

```php
[
    'NAME_1' => 'Name 1 Value',
    'NAME_2' => 'Name 2 Value',
]
```